src/server/api.ts

```diff
@@ -170,7 +180,8 @@ export const addApiToServer = ({ app, db, games, lobbyConfig }) => {
     }
 
     gameMetadata.players[playerID].name = playerName;
-    const playerCredentials = gameMetadata.players[playerID].credentials;
+    const playerCredentials = await lobbyConfig.generateCredentials(ctx);
```

Comment:
Should we also pass `playerID` to `generateCredentials` (in case it wants to do something player specific)?

Reply:
Potentially. It is in theory already available as `ctx.request.body.playerID`, but there’s probably an argument for some handy aliases. Anything else you can think of that might be useful?

Reply:
That's the only one that comes to mind at the moment.

Reply:
I'm also happy leaving it out at the moment until we see a use case for it.  I'll let you make the call.